extensions/qrcode-generator/package.json

```diff
@@ -46,6 +69,8 @@
   "dependencies": {
     "@raycast/api": "^1.77.0",
     "@raycast/utils": "^1.16.0",
+    "mac-open": "^0.1.3",
```

Comment:
This dependency was last updated [10 years ago](https://github.com/weareinteractive/node-mac-open) and is unmaintained.

You can probably use Raycast's `showInFinder` utility function:
https://developers.raycast.com/api-reference/utilities#showinfinder

```diff
- open(path);
+ showInFinder(path);
```